tests/Main.elm

```diff
@@ -0,0 +1,13 @@
+port module Main exposing (..)
```

Comment:
@cuducos I tried to follow your another (awesome) project, but I faced some errors, then I choose `elm-test` library to handle it. It's a generated file. I'll learn more about ports and try to remove it if necessary.

Thanks.